tests/register/available.test

```diff
@@ -0,0 +1,34 @@
+# intended behavior of --available flag
+# register: shows the smallest future balance
+#   (with --period: within the given period)
+
+# 1. --available causes smallest future balance to be displayed
+hledger -f - register --available a
+<<<
+2018/1/1 x
+  a  1
+  b
+
+2018/1/2 y
+  a  -0.5
+  b
+
+2018/1/3 x
+  a  1
+  b
+
+2018/1/4 z
+  a  -1.25
+  b  
+
+2018/1/5 x
+  a   1
+  b
+
+2018/1/6 y
+  a   -0.5
+  b
+>>>
+2018/01/01 x                    a                             1.00          0.25
+2018/01/05 x                    a                             1.00          0.75
```

Comment:
Why we get two lines for same account here having different values `0.25` and ` 0.75`?
I expected only one with date and minimum value (even if it is negative honoring the sign).

Reply:
I had the same question. It reports all the minima (of a curve plotting balance over time), not just the lowest minimum.